srcopsmetrics/iterator.py

```diff
@@ -69,7 +69,7 @@ def run(self):
             for idx, entity in enumerate(self.entity.analyse(), 1):
                 self.knowledge_updated = True
 
-                remaining = github.rate_limiting[0]
+                remaining = github.get_rate_limit().core
```

Comment:
this change produced errors: 

```
INFO:srcopsmetrics.bot_knowledge:########################
INFO:srcopsmetrics.bot_knowledge:DependencyUpdate inspection
INFO:srcopsmetrics.entities.tools.storage:Loading knowledge from Ceph
INFO:srcopsmetrics.entities.interface:No previous knowledge of type DependencyUpdate found
INFO:srcopsmetrics.iterator:-------------DependencyUpdate Analysis-------------
INFO:srcopsmetrics.iterator:Cached knowledge could not be saved
Traceback (most recent call last):
  File "srcopsmetrics/cli.py", line 129, in <module>
    cli()
  File "/home/fmurdaca/.local/share/virtualenvs/mi-_yZ92-Ga/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/fmurdaca/.local/share/virtualenvs/mi-_yZ92-Ga/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/fmurdaca/.local/share/virtualenvs/mi-_yZ92-Ga/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/fmurdaca/.local/share/virtualenvs/mi-_yZ92-Ga/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "srcopsmetrics/cli.py", line 111, in cli
    analyse_projects(projects=tupled_repos, is_local=is_local, entities=entities)
  File "/home/fmurdaca/work/aicoe/mi/srcopsmetrics/bot_knowledge.py", line 90, in analyse_projects
    github_repo=github_repo, project_path=project_path, entity_cls=entity, is_local=is_local
  File "/home/fmurdaca/work/aicoe/mi/srcopsmetrics/github_knowledge.py", line 187, in analyse_entity
    analysis.run()
  File "/home/fmurdaca/work/aicoe/mi/srcopsmetrics/iterator.py", line 74, in run
    if remaining <= API_RATE_MINIMAL_REMAINING:
TypeError: '<=' not supported between instances of 'Rate' and 'int'
```

Reply:
nice catch!
i fixed it so it is now relevant with github utc time, too